charts/rstudio-connect/Chart.yaml

```
@@ -1,14 +1,38 @@
 name: rstudio-connect
-description: Kubernetes deployment for RStudio Connect
-version: 0.2.0
+description: Official helm chart for RStudio Connect
```

```suggestion
description: Official Helm chart for RStudio Connect
```

Nit - Helm should be capitalized (proper noun).